config-ui/src/components/blueprints/create-workflow/DataTransformations.jsx

```diff
@@ -136,17 +109,47 @@ const DataTransformations = (props) => {
   )
   const [activeEntity, setActiveEntity] = useState()
 
-  const transformationHasChanged = useCallback(
+  const checkCurrentTransformationIsChanged = useCallback(
```

Comment:
Why wouldn't `checkCurrentTransformationIsChanged` and `changeCurrentTransformationSettings` be defined in the Transformation Manager?

Reply:
`Has` instead of `Is` would also read better.

```suggestion
  const checkCurrentTransformationHasChanged = useCallback(
```

Reply:
All are acceptable. I write it here because write related code in small scope instead of passing it through many levels. If you insist, I'll move it into hook.

Reply:
Yea I think these are key functions of the DSM / Transformation Manager. It would be better encapsulated in the Hook for consistency. We'll try to adapt some Contexts in the future to possibly help with our nested props being passed to many levels.

Reply:
moved